codemagic.yaml

```diff
@@ -37,6 +39,9 @@ workflows:
         gsutil -h "Content-Type:application/x-wheel+zip" \
                -h "Cache-Control:no-cache,max-age=0" \
                cp -a public-read "$SRC_PATH" "$DST_URL"
+
```

Comment:
Perhaps do it as a script step with either a comment or echo statement in the header for better understanding what is being done here?